hivemind/utils/grpc.py

```diff
@@ -160,6 +161,46 @@ def top(self) -> ValueError:
         raise ValueError(f"Please use {self.__class__.__name__}.get_stub to get or create stubs")
 
 
+def quantile_encode_torch_approx(weight, n_bits: int):
```

Comment:
What is the return type of this function? Also, it's probably best to rename the first argument to something more general (like `tensor`)